unstable/wlr-window-list.xml

```diff
@@ -0,0 +1,148 @@
+    AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+    ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
+    THIS SOFTWARE.
+  </copyright>
+
+  <interface name="zwlr_window_manager_v1" version="1">
+    <description summary="list and control opened apps">
+      The purpose of this protocol is to enable the creation of taskbars
+      and docks by providing them with a list of opened applications and
+      letting them request certain actions on them, like maximizing, etc.
+
+      After a client binds the zwlr_window_manager_v1, each opened window
+      will be sent via the window_created signal.
+    </description>
+
+    <event name="window_created">
+      <description summary="a window has been created">
+        This event is emitted whenever a new toplevel window is created.
+      </description>
+      <arg name="window" type="new_id" interface="zwlr_window_v1"/>
+    </event>
+
+    <request name="destroy" type="destructor"/>
+  </interface>
+
+  <interface name="zwlr_window_v1" version="1">
+    <description summary="an opened window">
+      A zwlr_window_v1 object represents an opened window.
+      Each app may have multiple opened windows.
+
+      Each window has a list of outputs it is visible on, conveyed to the client
+      with the enter_output and leave_output events.
+    </description>
+
+    <event name="title">
+      <description summary="title change">
+        This event is emitted whenever the title of the window changes
+        and immediately after the creation of the zwlr_window_v1
+      </description>
+      <arg name="title" type="string"/>
+    </event>
+
+    <event name="app_id">
+      <description summary="app-id change">
+        This event is emitted whenever the app-id of the window changes
+        or after it is created (the initial title)
+      </description>
+      <arg name="app_id" type="string"/>
+    </event>
+
+    <event name="enter_output">
+      <description summary="window entered an output">
+        This event is emitted whenever the window becomes visible on
+        the given output. A window may be visible on multiple outputs.
+      </description>
+      <arg name="output" type="object" interface="wl_output"/>
+    </event>
+
+    <event name="leave_output">
+      <description summary="window left an output">
+        This event is emitted whenever the window stops being visible on
+        the given output. It is guaranteed that an entered-output event
+        with the same output has been emitted before this event.
+      </description>
+      <arg name="output" type="object" interface="wl_output"/>
+    </event>
+
+    <event name="virtual_desktop">
+      <description summary="window was moved to a virtual desktop">
+        This event is emitted with the initial virtual desktop the window
+        is on and whenever it is moved to a different virtual desktop later.
+      </description>
+      <arg name="desktop" type="string"/>
+    </event>
```

Comment:
This probably belongs in a separate protocol. It's a subtle issue.

https://github.com/swaywm/wlr-protocols/issues/10

Reply:
okay, will remove it for now